iroh-gossip/src/proto/state.rs

```diff
@@ -196,6 +196,11 @@ impl<PI: PeerIdentity, R: Rng + Clone> State<PI, R> {
             .unwrap_or(false)
     }
 
+    /// Returns the maximum message size configured in the gossip protocoli.
```

Comment:
```suggestion
    /// Returns the maximum message size configured in the gossip protocol.
```
:broccoli: :broccoli: :broccoli: